src/Microsoft.Azure.WebJobs.Extensions.EventHubs/Config/EventHubOptions.cs

```diff
@@ -60,6 +61,8 @@ public int BatchCheckpointFrequency
             }
         }
 
+        public InitialOffsetOptions InitialOffsetOptions { get; set; }
+
```

Comment:
Maybe in can be just a string:
public string InitialOffset { get; set; } // possible values ["start", "end", {DateTime}]

Reply:
I don't like assuming that if an option is not "start" or "end" it is a date time. Can make expanding the set of options in the future to add non-string options harder.  This was is more structured and allowed for multiple values to be specified for a given option if needed in the future.